src/_bentoml_sdk/images.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import annotations
```

Comment:
Oh final thing, we should also expose this in `bentoml/__init__.py` so that `bentoml.Image` works